src/jit/compiler.cpp

```diff
@@ -4815,8 +4815,8 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, JitFlags
     m_pLinearScan = getLinearScanAllocator(this);
 
     /* Lower */
-    Lowering lower(this, m_pLinearScan); // PHASE_LOWERING
-    lower.Run();
+    m_pLowering = new (this, CMK_LSRA) Lowering(this, m_pLinearScan); // PHASE_LOWERING
```

Comment:
It would be nice if we could run liveness before lowering but I suppose it's problematic to ensure that lowering preserves liveness. As is now we end up passing some dead code through lowering, it only gets removed during liveness.